contrib/datastore/file/gcp/gcp.go

```diff
@@ -30,7 +44,44 @@ type GcpAdapter struct {
 }
 
 func (g GcpAdapter) CreateFileShare(ctx context.Context, fs *file.CreateFileShareRequest) (*file.CreateFileShareResponse, error) {
-	panic("implement me")
+	instanceName := fs.Fileshare.Name + "-instance"
+	labels := make(map[string]string)
+	for _, tag := range fs.Fileshare.Tags {
+		labels[tag.Key] = tag.Value
+	}
+
+	instance := &gcpfilev1.Instance{
+		Description: fs.Fileshare.Description,
+		Tier: fs.Fileshare.Metadata.Fields[Tier].GetStringValue(),
+		FileShares: []*gcpfilev1.FileShareConfig{
+			{
+				Name:       fs.Fileshare.Name,
+				CapacityGb: fs.Fileshare.Size / utils.GB_FACTOR,
+			},
+		},
+		Networks: []*gcpfilev1.NetworkConfig{
+			{
+				Network:         DefaultNetwork,
+				Modes:           []string{InternetProtocolModeIpv4},
+			},
+		},
+		Labels: labels,
+	}
+
+	log.Infof("Starting CreateInstance on cloud backend")
```

Comment:
Redundant log message, next line is fine.

Reply:
First one is specifying the GCP backend operation. The next line to log the instance object.
I will update the log level to debug for next line.

Reply:
Ok